apps/settings/locales/settings.en-US.properties

```diff
@@ -303,7 +303,7 @@ message-read-reports-details = Request read report for each message sent
 # force the proper directionality of parentheses at the end of the string
 # regardless of the direction of the surrounding layout. Remove them only
 # if the translation puts them between other text.
-mmsTitle.innerHTML= &#x202a;Multimedia Messaging (MMS)&#x202c;
+mmsTitle= &#x202a;Multimedia Messaging (MMS)&#x202c;
```

Comment:
@stasm - in bug 1139031 you recommended this solution. If I'm correct that is either not needed (because we do inject it in l10n.js for fallback languages) or it should be affecting all <h2> elements and should use CSS dir:auto instead.

Am I correct? I'd like to remove those characters from localization files